### PR TITLE
Adapt existing language into explainer template

### PR DESCRIPTION
**Pull request by arichiv**
The structure isn't quite aligned with the needs of an explainer, let's try to massage it without introducing significantly different language/concepts.

### CODE REVIEW

**Comment by arichiv** [comment]
@johannhof could use review if you have time

File: README.md [hunk 1]
```diff
@@ -6,15 +6,23 @@
...
-The ability of windows to communicate with each other is critical to current web infrastructure, especially for payment and login models. Whatever changes are pursued in this space must proceed with care to avoid breaking the web.
+## Goals
+
+1. Ensure inter-window communication isn't disrupted in a way detrimental to user experience (especially around payment and login functions).
+2. Prevent cross-storage-bucket information leaks that enable cross-site tracking.
+
+## Non-Goals
+
+1. Address all breakage resulting from storage partitioning and window isolation.
+2. Force adoption of FedCM and other purpose-built APIs.
```
Comment (johannhof) [inline README.md]: Might want to add some flavor to this, e.g.

```suggestion
2. Force adoption of FedCM and other purpose-built APIs. While we encourage usage of these APIs, developers should continue to be able to build cross-site user experiences with popups (taking into account additional friction from end-user privacy protections).
```

File: README.md [hunk 2]
```diff
@@ -183,6 +179,12 @@ If we only force the partitioning (or block opener references) for windows opene
 
 This would break a huge amount of the web.
 
-## Chrome Timeline
+## Privacy & Security Considerations
```
Comment (johannhof) [inline README.md]: Aren't these more like "user experience considerations"?

File: README.md [hunk 3]
```diff
@@ -6,15 +6,23 @@
...
+The former proposal is primarily concerned with user privacy (e.g., preventing the average user from having their browsing unknowingly tracked in order to serve ads) whereas the latter proposal is primarily concerned with user security (e.g., ensuring even a targeted user cannot have information leaked cross-site). A way to bridge the gap between these needs should be considered to ensure both constraints can be met without disrupting user experience.
 
-The ability of windows to communicate with each other is critical to current web infrastructure, especially for payment and login models. Whatever changes are pursued in this space must proceed with care to avoid breaking the web.
+## Goals
+
+1. Ensure inter-window communication isn't disrupted in a way detrimental to user experience (especially around payment and login functions).
+2. Prevent cross-storage-bucket information leaks that enable cross-site tracking.
```
Comment (johannhof) [inline README.md]: nit: it's easier to read if you swap these two items

Comment (johannhof) [inline README.md]:
```suggestion
2. Prevent cross-storage-bucket information leaks from window openers that enable cross-site tracking.
```

**Review by johannhof — APPROVED** [review]
LGTM!

**Pull request merged**